chapters/Chapter 1 — Untitled.md

```diff
@@ -49,14 +49,76 @@ Something wet was pooling at the bottom of her helmet’s goggles. Right: she wa
+
+He saw her gun. He waved his hands, tried to wriggle away. His hands scraped at the nothingness— then stopped. The bullet pierced his heart. Frothy blood flowed into the vacuum, boiling and freezing.
+
+Leanne’s eyes returned to the hodgepodge of a ship that had attacked hers. Its spin had stopped. Its engines were only now about to fire. Their Alts were definitely flying it—only Alts were so thorough. Humans would have left already, checklists be damned.
+
+Leanne brought her feet up and nudged the field generator. It pulled her towards the other ship, even as it began to accelerate.
+
+She landed on its hull. Her knees buckled and her head spun. The landing had been a bit harder than she had intended.
+
+The cockpit wasn’t too far. One foot, then the other, she moved. She blinked her eyes, but it didn’t quite bring the brightness back. There wasn’t much to steady herself with. She could taste the blood on her lips. She took deep breaths. She’d be fine. Each breath was fresh from her tanks, rich in oxygen.
+
+Leanne took an explosive from her belt, and planted it on the hull, just behind the cockpit windows. The windows would be the strongest part; the joints holding them the weakest. 
+
+She stepped back. Entered a code on her wrist. It seemed low tech, but that was the way with most weapons. More advanced technology could only be controlled by Alts.
+
+The explosion ripped the front of the cockpit off her attackers’ still-accelerating ship. Leanne hit the hull just before the debris shot over her.
+
+Leanne’s other attacker flew out of the hole. A bullet followed her. Leanne supposed it hadn’t been necessary: the other attacker wasn’t wearing a suit. She’d be out in thirty seconds, and dead in another thirty, unless her Alt could pull off a miracle. But Leanne’s rule was Leanne’s rule.
+
+Leanne forced herself to watch until the end. Her aim hadn’t been perfect. She’d hit the woman’s neck instead of her head.
+
+The woman’s twitching slowed. The fury in her boiling eyes dimmed. 
+
+Leanne took a deep breath, and crawled into her attacker’s ship. Her head spun as she landed in the remains of the cockpit. 
+
+She checked the feeds from the cameras. It appeared she was alone, or alone enough. She rifled through the console. Her attackers, like her, were assassins. They should have a card for the job. It would have all the pertinent details, and they’d turn it in with proof of death. 
+
+It was unlikely there’d be anything on the card to trace back to the submitter; assassins always worked through an Arranger, and even the Arranger rarely knew who hired them. But Leanne searched all the same, whether out of habit or hunch.
+
+The card was small. There was a little yellow insignia on the upper-left of a diving eagle—the Leesome Arrangers, if Leanne remembered correctly. There was little information on it, besides a picture of her and her ship, and a location she once frequented years ago. It was extremely impressive that they found her at all. 
```

Comment:
Something cryptographically important about the cards. Are they proof you took the job, without which you cannot get payment?

Okay, if so:

- Cards are expensive
- Any assassin should usually have just one at a time, because they are expensive: don't want assassins just collecting as many as they can and hoping they get lucky. Too pricey.

"The cards was small. There was a little yellow insignia on the upper-left of a diving eagle: the Leesomes, if Leanne remembered correctly. They were arrangers. There was little information on the card, besides a picture of her and her ship, and a location she once frequented years ago. It was extremely impressive that her attackers had found her at all.

"There was another card. Same insignia. It was odd. Although arrangers would give the same card to as many assassins as would take them, they would rarely give one assassin more than one card at once. Leanne always assumed it because the cards themselves were expensive due to whatever cryptographic chips were on them, and that the arrangers didn't want the assassins hoarding the cards hoping they'd get lucky and kill one of the targets eventually."